variable/technology/technologies.yaml

```diff
@@ -522,570 +521,3 @@ Investment|Energy Supply|Liquids|Oil:
-   description: inertia provided to the system by a Photovoltaic power plant
-   unit: second
-   
-Rate Frequency Containment Reserve|Electricity|Solar:
-   description: Percentage of the Maximum Active Power that can be devoted to Frequency Containment Reserve
-                for a Photovoltaic power plant
-   unit: %
-   
-Rate Automatic Frequency Restoration Reserve|Electricity|Solar:
-   description: Percentage of the Maximum Active Power that can be devoted to Automatic Frequency Restoration Reserve
-                for a Photovoltaic power plant
-   unit: %
-   
-# description of Wind Onshore power plants
-Number of Units|Electricity|Wind|OnShore:
-   description: Number of units of kind Wind Onshore power plant
-   unit: 
-   
-Maximum Active power|Electricity|Wind|OnShore:
-   description: Maximum active power of one Wind Onshore power plant
-   unit: MW
-
-LoadFactor|Electricity|Wind|OnShore|Profile:
-   description: Timeserie of hourly loadfactors for Onshore Windpower
-   unit: %
-   
-Minimum Active power|Electricity|Wind|OnShore:
-   description: Minimum active power of one Wind Onshore power plant
-   unit: MW
-   
-Inertia|Electricity|Wind|OnShore:
-   description: inertia provided to the system by a Wind Onshore power plant
-   unit: second
-   
-Rate Frequency Containment Reserve|Electricity|Wind|OnShore:
-   description: Percentage of the Maximum Active Power that can be devoted to Frequency Containment Reserve
-                for a Wind Onshore power plant
-   unit: %
-   
-Rate Automatic Frequency Restoration Reserve|Electricity|Wind|OnShore:
-   description: Percentage of the Maximum Active Power that can be devoted to Automatic Frequency Restoration Reserve
-                for a Wind Onshore power plant
-   unit: %
-   
-# description of Wind Offshore power plants
-Number of Units|Electricity|Wind|OffShore:
-   description: Number of units of kind Wind OffShore power plant
-   unit: 
-   
-Maximum Active power|Electricity|Wind|OffShore:
-   description: Maximum active power of one Wind OffShore power plant
-   unit: MW
-
-LoadFactor|Electricity|Wind|OffShore|Profile:
-   description: Timeserie of hourly loadfactors for Offshore Windpower
-   unit: %
-   
-Minimum Active power|Electricity|Wind|OffShore:
-   description: Minimum active power of one Wind OffShore power plant
-   unit: MW
-   
-Inertia|Electricity|Wind|OffShore:
-   description: inertia provided to the system by a Wind OffShore power plant
-   unit: second
-   
-Rate Frequency Containment Reserve|Electricity|Wind|OffShore:
-   description: Percentage of the Maximum Active Power that can be devoted to Frequency Containment Reserve
-                for a Wind OffShore power plant
-   unit: %
-   
-Rate Automatic Frequency Restoration Reserve|Electricity|Wind|OffShore:
-   description: Percentage of the Maximum Active Power that can be devoted to Automatic Frequency Restoration Reserve
-                for a Wind OffShore power plant
-   unit: %
```

Comment:
All this was moved to another file, right? If yes, perfect

Reply:
Yeah, I moved all of them to power-plant.yaml.
And, the classification per technology is preserved.